sql/sqlite/inspect.go

```diff
@@ -715,7 +727,7 @@ const (
 	databasesQuery     = "SELECT `name`, `file` FROM pragma_database_list() WHERE `name` <> 'temp'"
 	databasesQueryArgs = "SELECT `name`, `file` FROM pragma_database_list() WHERE `name` IN (%s)"
 	// Query to list database tables.
-	tablesQuery = "SELECT `name`, `sql` FROM sqlite_master WHERE `type` = 'table' AND `name` NOT LIKE 'sqlite_%' AND `name` NOT LIKE 'libsql_%'"
+	tablesQuery = "SELECT sqlite_master.name, sqlite_master.sql, `wr`, `strict` FROM sqlite_master join(pragma_table_list(sqlite_master.name)) WHERE sqlite_master.type = 'table' AND sqlite_master.name NOT LIKE 'sqlite_%' AND sqlite_master.name NOT LIKE 'libsql_%'"
```

Comment:
Use `pragma_table_list` only in new versions of SQLite. See how we do it in MySQL: https://github.com/ariga/atlas/blob/master/sql/mysql/inspect.go#L695-L696 (we have 2 different statements).

Reply:
I played with it a bit and it seems to be backwards compatible. since strict was also introduced in version 3.37, databases created by older client will always show strict=0.